C0T0/Classes/ApiService/ApiService.swift

```diff
@@ -44,12 +46,11 @@ public class ApiService {
             urlRequestBuilder: StandardURLRequestBuilder(host:configuration.host),
             urlRequestSender: StandardURLRequestSender(urlSession: URLSession.shared),
             converter: StandardURLRequestSenderErrorConverter(),
-            configuration: ApiDataSourceConfiguration()
+            configuration: ApiDataSourceConfiguration(keyDecodingStrategy: configuration.keyDecodingStrategy, loggingEnabled: configuration.loggingEnabled)
```

Comment:
No changes in `ApiDataSourceConfiguration`?

Reply:
Both arguments were set by default